examples/DynaFlow/description/DynaFlowExamples.tex

```diff
@@ -211,6 +211,91 @@ \section{Load restoration (LoadRestoration)}
 
 When the restoration is not complete, the active power of the load is below the maximum transferrable power. As a consequence, the simulation succeeds.
 
+\chapter{WSCC9}
+
+The WSCC 9-bus system represents a simple approximation of the Western System Coordinating Council (WSCC) to an equivalent system with nine buses and three generators.
+It was notably published in Anderson and Fouad’s book ‘Power System Control and Stability’ for the first time in 1980.
+
+\section{Test case description}
+
+The WSCC 9-bus system has 9 buses, 3 generators, 3 transformers, 6 lines and 3 loads.  \\
+There are four voltage levels in the test case: 230 kV, 18 kV, 16.5 kV and 13.8 kV. The inner
+part is the 230 kV part. The generators are at 18 kV, 16.5 kV and 13.8 kV.
+
+\begin{figure}[H]
+  \includegraphics[width=\textwidth]{WSCC9Bus.jpg}
+  \caption{WSCC 9-bus system diagram}
+\end{figure}
+
+\subsection{Initial Conditions}
+
+The reference angle for the load flow is set at bus n°1. The initial conditions for each generator
+are:
+\begin{center}
+\begin{tabular}{|c|c|c|c|c|}
+  \hline
+  Generator & P (MW) & Q (Mvar) & U (kV) & $\Theta$ (°) \\
+  \hline
+  1 & 71.94 & 24.05 & 15.60 & 0.00\\
+  2 & 163.00 & 14.44 & 18.00 & 9.66\\
+  3 & 85.00 & -3.65 & 13.80 & 4.77\\
+  \hline
+\end{tabular}
+\end{center}
+
+\subsection{Models}
+
+\subsubsection{Synchronous machines}
+
+All the generators and the synchronous condensers are modeled as GeneratorPV: it means that they can adjust their Q to maintain the voltage equal to a target and their active power is modified by the frequency regulation model through the signal N variable.
```

Comment:
Is there any synchronous condenser in this test case ?

Reply:
No, copy/paste mistake!